geojson_pydantic/geometries.py

```diff
@@ -52,6 +57,41 @@ class MultiPolygon(_GeometryBase):
     type: str = Field("MultiPolygon", const=True)
     coordinates: List[List[List[Coordinate]]]
 
+
 class GeometryCollection(BaseModel):
     type: str = Field("GeometryCollection", const=True)
-    geometries: List[Union[Point, MultiPoint, LineString, MultiLineString, Polygon, MultiPolygon]]
+    geometries: List[
+        Union[Point, MultiPoint, LineString, MultiLineString, Polygon, MultiPolygon]
+    ]
+
+
+Geometry = Union[Point, MultiPoint, LineString, MultiLineString, Polygon, MultiPolygon]
+
+
+def parse_geometry_obj(obj) -> Geometry:
+    """
+    `obj` is an object that is supposed to represent a GeoJSON geometry. This method returns the
+    reads the `"type"` field and returns the correct pydantic Geometry model.
+    """
+    if "type" not in obj:
+        raise ValidationError(
+            [
+                ErrorWrapper(ValueError("Missing 'type' field in geometry"), "type"),
```

Comment:
I _think_ this is a decent way to report a ValidationError. Maybe good enough for a first try and possible to iterate on it if needed?